packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -77,6 +77,9 @@ class AccessLegBody extends Component<Props, State> {
     } = this.props;
     const { expanded } = this.state;
 
+    const hideDrivingDirections =
+      config?.itinerary?.hideDrivingDirections && leg.mode === "CAR";
```

Comment:
Quite frankly, I hate passing the config like that, but because that's already done elsewhere in this package I won't hold this against this PR.

Reply:
Hi Binh, I'm also doing a configurable change within otp-ui. What is the preferred method to pass the config?

Reply:
I'm guessing it's preferred to just pass the individual options in rather than use the whole config object.